ts/webui/src/static/function.ts

```diff
@@ -4,6 +4,17 @@ import { IContextualMenuProps } from '@fluentui/react';
 import { MANAGER_IP } from './const';
 import { MetricDataRecord, FinalType, TableObj, Tensorboard } from './interface';
 
+function getPrefix(): string | undefined {
+    const pathName = window.location.pathname;
+    let newPathName = pathName;
+
+    if (pathName.endsWith('/oview' || '/detail' || '/experiment')) {
```

Comment:
Shouldn't it be:

`pathName.endsWith('/oview') || pathName.endsWith('/detail') || pathName.endsWith('/experiment')`

Reply:
will change it at once! it's my oversight. Thanks!

Reply:
had been fixed